protected/humhub/modules/admin/models/forms/ApproveUserForm.php

```diff
@@ -228,6 +257,36 @@ public function send(): bool
         return $mail->send();
     }
 
+    /**
+     * Sets the subject and message attribute texts for user decline
+     * @return void
+     */
+    public function setSendMessageDefaults()
+    {
+        Yii::$app->i18n->setUserLocale($this->user);
+
+        /** @var Module $module */
+        $module = Yii::$app->getModule('user');
+
+        $this->subject = Yii::t('AdminModule.user',
+            'About the account request for \'{displayName}\'.',
```

Comment:
@Semir1212 Can you please review, this mail subject?

Reply:
"Your pending registration for \'{displayName}\"